package.json

```diff
@@ -15,7 +15,7 @@
   },
   "repository": {
     "type": "git",
-    "url": "git+https://github.com/blueqaus/r2-form.git"
+    "url": "blueqaus/r2-form"
```

Comment:
Umm possibly, I was just having troubles early on as the url wasn't playing nice with my npm when I tried to push.
 [https://docs.npmjs.com/files/package.json#repository](https://docs.npmjs.com/files/package.json#repository) 
I just looked at the npm docs and it didn't have it in the format you had given it so I thought it was an error. Not sure if the old url will still cause issue.